Snake/DatabaseLayer/Database.cs

```diff
@@ -1,36 +1,74 @@
-﻿using System.IO;
+﻿using System.Data;
+using System.Data.SqlClient;
+using System.Collections.Generic;
 
 namespace DatabaseLayer
 {
     public class Database
     {
+        private const int MaxNoOfScores = 5;
+        private SqlConnection sqlConnection;
+        private SqlCommand cmd;
+        private SqlDataReader sqlDataReader;
+        private string mazeLevel;
+
+        public Database(string mazeLevel)
+        {
+            sqlConnection = new SqlConnection("Server= localhost; Database= SnakeHighScores; Integrated Security=True;");

```

Comment:
I would not be creating any of this stuff in the constructor - it's OK to get a new connection and command in each method (especially once we incorporate using statements)